openpype/modules/sync_server/sync_server_module.py

```diff
@@ -402,6 +396,200 @@ def get_remote_site(self, project_name):
 
         return remote_site
 
+    def get_configurable_items(self, scope=None):
+        """
+            Returns list of items that could be configurable for all projects.
+
+            Could be filtered by 'scope' argument (list)
+
+            Args:
+                scope (list of utils.EditableScope)
+
+            Returns:
+                (dict of list of dict)
+                {
+                    siteA : [
+                        {
+                            key:"root", label:"root",
+                            "value":"{'work': 'c:/projects'}",
+                            "type": "dict",
+                            "children":[
+                                { "key": "work",
+                                  "type": "text",
+                                  "value": "c:/projects"}
+                            ]
+                        },
+                        {
+                            key:"credentials_url", label:"Credentials url",
+                            "value":"'c:/projects/cred.json'", "type": "text",
+                            "namespace": "{project_setting}/global/sync_server/
+                                     sites"
+                        }
+                    ]
+                }
+        """
+        editable = {}
+        applicable_projects = list(self.connection.projects())
+        applicable_projects.append(None)
+        for project in applicable_projects:
+            project_name = None
+            if project:
+                project_name = project["name"]
+
+            items = self.get_configurable_items_for_project(project_name,
+                                                            scope)
+            editable.update(items)
+
+        return editable
+
+    def get_local_settings_schema_for_project(self, project_name):
+        """Wrapper for Local settings"""
+        return self.get_configurable_items(project_name, EditableScopes.LOCAL)
```

Comment:
Probably should be `get_configurable_items_for_project`

Reply:
Yes, fixed.